src/app/app.constants.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+angular.module('esn-frontend-inbox-calendar')
```

Comment:
We have all our modules named like`esn.something`, can you do the same?

Reply:
Renamed